tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -1224,10 +1224,10 @@ STATIC_ASSERT(__cpp_lib_quoted_string_io == 201304L);
 #if _HAS_CXX23 && !defined(__EDG__) // TRANSITION, EDG concepts support and GH-1814
 #ifndef __cpp_lib_ranges
 #error __cpp_lib_ranges is not defined
-#elif __cpp_lib_ranges != 201911L
-#error __cpp_lib_ranges is not 201911L
+#elif __cpp_lib_ranges != 202106L
+#error __cpp_lib_ranges is not 202106L
 #else
-STATIC_ASSERT(__cpp_lib_ranges == 201911L);
+STATIC_ASSERT(__cpp_lib_ranges == 202106L);

```

Comment:
also possibly preliminary

Reply:
Ditto this has landed in the working draft.